GloVe/download_glove_840b_300d.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+echo "Downloading GloVe Common Crawl (840B tokens, 2.2M vocab, cased, 300d vectors, 2.03 GB download)"
+wget http://nlp.stanford.edu/data/glove.840B.300d.zip
```

Comment:
This is a great idea. I committed the MD5 checksum and if the file exists then do not download it.